### ISSUE TITLE
Desktop App icon is absent in window switcher (Alt + Tab) in GNOME Shell 3.24.3

### ISSUE THREAD

**Issue by sancho-one** [standardnotes/desktop#101]
SN Notes' icon is displayed in 1. Window Laucher normally (see screenshot), but it's broken (it's displayed "broken image" icon) in 2. gnome window switcher (Alt + Tab, unfortunately I can't figure out how to take screenshot of window switcher so I just send you a random image of window switcher):

1. ![screenshot from 2017-10-09 19-11-18](https://user-images.githubusercontent.com/32115115/31363808-9fa155f2-ad26-11e7-8b6c-86f7fdf1be80.png)
2. ![screenshot_15](https://user-images.githubusercontent.com/32115115/31363809-a228b112-ad26-11e7-93bd-74b04fa1f6c8.png)

**Comment by mobitar** [comment]
If you aren't already doing this, after installing the AppImage, you should launch it via the Launcher and not the AppImage. I've found this to clear up the icon issue on Linux before.

Or perhaps you can try the new v2 beta and see if that makes a difference: https://github.com/standardnotes/desktop-beta/releases/tag/v2.0.0-beta3

**Comment by sancho-one** [comment]
Yes, of course I launch it via Laucher as it's obvious benefit of having an app installed in your system. ;)
Thanks, I'll try v2 beta today.

**Comment by sancho-one** [comment]
@mobitar I've just downloaded and installed AppImage v2 beta. Unfortunately, the problem with gnome window switcher persists. Maybe, it needs an bigger size icon?

PS. v2 is great, especially with Archived tag. Nevertheless, it didn't work. I couldn't tag any notes with it. :(

**Comment by mobitar** [comment]
What do you mean you couldn't tag any notes with it?

**Comment by sancho-one** [comment]
@mobitar no, I couldn't. I write tag "Archived", hit enter and nothing.

**Comment by mobitar** [comment]
Could you try with a different tag name? And opening console to see if any JavaScript errors occur?

**Comment by sancho-one** [comment]
I don't know about inner structure of AppImage, but as for traditional Gnome 3 icons: probably place the appropriate icon in the scalable or 96 subfolder of /usr/share/themes/gnome or local user folder.

![img_20171010_164535](https://user-images.githubusercontent.com/32115115/31412527-0f498cb2-addb-11e7-8fd8-e3a8d2ec32d6.jpg)

**Comment by sancho-one** [comment]
@mobitar Well, all other tags work fine. When I add "Archived" tag, it just creates new tag name "Archived". It looks like it didn't recognize "system" tag Archived. See screenshot:

![screenshot from 2017-10-10 16-55-44](https://user-images.githubusercontent.com/32115115/31412748-fc147c96-addb-11e7-981f-f9f8d504f172.png)

**Comment by mobitar** [comment]
To archive a note, press “Menu” on the note, then click Archive. Archive is the last tag in the tag list on the left, and it’s grayed out. It’s not a real tag. It’s a system tag.

**Comment by sancho-one** [comment]
Ah great, it worked. Is there an option to toggle show/hide archived notes
withing tag? I.e. show all my notes with tag "Shopping" which are not
archived / show all my notes with tag "Shopping" archived and not archived
(like Inbox/All mail in Gmail)?

**Comment by sancho-one** [comment]
Pin function works great, it's a real advancement as for v2. I guess I have
no need in "Bookmarks" tag any more!

**Comment by mobitar** [comment]
> Is there an option to toggle show/hide archived notes
withing tag?

Not currently. But this wouldn't be too hard to do in an update.

> Pin function works great, it's a real advancement as for v2.

Glad to hear it :) Be sure to use the beta version of the mobile apps if you want pins and archives to transfer over. More info on twitter.com/standardnotes.

**Comment by sancho-one** [comment]
@mobitar Great! Already installed. ;)
